containers/dotnet/definition-manifest.json

```diff
@@ -7,7 +7,12 @@
 		"tags": [
 			"dotnet:${VERSION}-${VARIANT}"
 		],
+		"architectures": ["linux/amd64", "linux/arm64"],
 		"variantTags": {
+			"6.0-focal": [
+				"dotnet:${VERSION}-6.0",
+				"dotnetcore:${VERSION}-6.0"
```

Comment:
am not 100% sure, but I don't think this `dotnetcore:<version>-6.0` container + tag is published, based on the [mcr list here](https://mcr.microsoft.com/v2/vscode/devcontainers/dotnetcore/tags/list) for 'vscode/devcontainers/dotnetcore'

Reply:
Good catch, thought I saw it but it seems to only go to 5

Reply:
Yeah - we kept it for older versions of C# for compatibility after the change. New ones like 6.0 we dropped it since it's not .NET Core.